test/DynamoCoreTests/IconsTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Dynamo.Models;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+using Dynamo.Wpf.Services;
+using DynamoShapeManager;
+using DynamoUtilities;
+
+using NUnit.Framework;
+
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Tests
+{
+    [TestFixture]
+    internal class IconsTests
+    {
+        private static string geometryFactoryPath = "";
+        private IEnumerable searchEntries;
+
+        static readonly string AssemblyPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        static readonly string AnyCPUPath = Directory.GetParent(AssemblyPath).FullName;
+        static readonly string binPath = Directory.GetParent(AnyCPUPath).FullName;
+        static readonly string DynamoPath = Directory.GetParent(binPath).FullName;
+
+        static readonly string ResourcePath = Path.Combine(DynamoPath, @"src\Resources");
```

Comment:
The above steps can be folded into a helper method `InitializeDirectoryPaths` so that we can assert that nothing changes in those assumptions. Perhaps it can do this:

```csharp
var assemblyDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
var relativeDir = Path.Combine(asmDir, @"..\..\..\src\Resources");
var absoluteDir = Path.GetFullPath(relativeDir);
Assert.IsFalse(string.IsNullOrEmpty(absoluteDir));
Assert.IsTrue(Directory.Exists(absoluteDir));
```